lib/NativeCode/DynamicLibraryLoaderHelper/NativeRender/pch.h

```diff
@@ -31,10 +31,12 @@
 #define ENABLE_DLL_BASED_EOS_CONFIG 1
 #define XAUDIO2_DLL_NAME "xaudio2_9redist.dll"
 
-#define EOS_SERVICE_CONFIG_FILENAME "EpicOnlineServicesConfig.json"
+#define EOS_WINDOWS_CONFIG_FILENAME "eos_windows_config.json"
+#define EOS_PRODUCT_CONFIG_FILENAME "eos_product_config.json"
+
 #define EOS_STEAM_CONFIG_FILENAME "eos_steam_config.json"
 #define EOS_LOGLEVEL_CONFIG_FILENAME "log_level_config.json"
 
-#define RESTRICT __restrict
+//#define RESTRICT __restrict
```

Comment:
Figure out why this was originally needed, and if there isn't a problem than remove it properly.